lib/std/jshttpcore.nim

```diff
@@ -0,0 +1,4 @@
+## Exports [jsformdata](jsformdata.html) and [jsheaders](jsheaders.html).
+
+import std/[jsformdata, jsheaders]
+export jsformdata, jsheaders
```

Comment:
What are the concrete technical benefits of doing this?, 
it was not designed to mimic `Httpclient` whatsoever.

If you prefer an API that feels like `Httpclient`, then copipaste the `jshttpclient` from https://github.com/juancarlospaco/nodejs/blob/main/src/nodejs/jshttpclient.nim#L169-L195

IMHO

Reply:
Why have `jsfetch` in the standard library if it's not meant to fill in for `httpclient`?

Reply:
To address the actual benefit: The benefit is that we don't need to worry about each import `jsfetch` might need to use, and just import one module and be done with it. I can have them both be exported in `jsfetch` instead.

Reply:
I dont know, core devs only allow me to keep it bare bones, thin wrapper around JS stuff.
See https://github.com/nim-lang/Nim/pull/17373#issue-831170015  :shrug: 

[Original PR was in 2019](https://github.com/nim-lang/Nim/pull/12531#issue-512915789)

Reply:
Ok well I don't even know if httpclient is meant to be kept in the standard library, and if it's only kept because CI & some tools use it. If we want to move it out then we don't need jsfetch either.

If we don't have `jshttpcore`, at the very least we should export these modules in `jsfetch` instead.

Reply:
Wrong place for this discussion IMO.